lib/nimbusRecipe.ts

```diff
@@ -14,7 +13,10 @@ import {
 } from "../lib/experimentUtils.ts";
 import { getExperimentLookerDashboardDate } from "./lookerUtils.ts";
 
-type NimbusExperiment = types.experiments.NimbusExperiment;
+const nimbusExperimentV7Schema = require("@mozilla/nimbus-schemas/schemas/NimbusExperimentV7.schema.json");
+type NimbusExperiment = typeof nimbusExperimentV7Schema.properties;
+type DocumentationLink =
+  typeof nimbusExperimentV7Schema.properties.documentationLinks;
```

Comment:
I think it would be helpful to put JSDoc comments here explaining why you're doing the type aliasing.